documentation/general/dash-sonic-hld.md

```diff
@@ -283,7 +283,17 @@ pl_sip_encoding          = Private Link encoding for IPv6 SIP transpositions; Fo
 pl_underlay_sip          = Underlay SIP (ST GW VIP) to be used for all private link transformation for this ENI
 ```
 ### 3.2.4 ACL
-  
+
+```
+DASH_PREFIX_TAG_TABLE:{{tag_name}}
+    "prefix_list": {{list of prefix}}
+```
+
+```
+tag_name                  = STRING; unique tag name
+addresses                 = list of ip prefixes ',' separated
```

Comment:
add a statement that says, "valid to have empty list of prefixes"

Reply:
Done